docs/Serverless.md

```diff
@@ -190,3 +191,73 @@ gcloud beta run deploy --image gcr.io/PROJECT-ID/APP-NAME --platform managed
 ```
 
 Your app will be accessible from the URL GCP provides.
+
+## Zeit Now
+
+[now](https://zeit.co/home) provides zero configuration deployment for
+Node.js applications. In order to use now, it is as simple as
+configuring your `now.json` file like the following:
+
+```json
+{
+  "builds": [
+    {
+      "src": "serverless.js",
+      "use": "@now/node",
+      "config": {
+        "helpers": false
+      }
+    }
+  ],
+  "routes": [
+    { "src": "/.*", "dest": "/serverless.js"}
+  ]
+}
+```
+
+Then, write a `serverless.js` like so:
+
+```js
+'use strict'
+
+const build = require('./index')
+
+const app = build()
+
+module.exports = async function (req, res) {
+  await app.ready()
```

Comment:
should we run `ready` at every request?

Reply:
AWS Lambda will reuse the same process multiple times. As a result, the promise returned by `ready()` will be resolved immediately. My check is that most of our overhead come Ajv startup, see the discussion in #1598.

Reply:
Thank you both for the clarification 👍